change-address/REPWRITERSPECS/BANNO.TRACKING8.FM

```diff
@@ -0,0 +1,134 @@
+[ File Name: BANNO.TRACKING8.FM
+  
+  Description: Batch PowerOn creates an FM report to update, 
+               expire or delete tracking type 8 records depending
+               on user selection.
+                
+  
+  Original Author: JuCarson 10/4/2022   R 2022.00
+
+  Details:  
+   Prompts:
+    Select Run Mode:
+      Expire All Tracking 8s       - PowerOn will expire all unexpired tracking
+                                     type 8 records found on any open account.
+
+      Delete All Tracking 8s       - PowerOn will delete all tracking type 8
+                                     records found on any open account.
+
+      Delete Banno-specific Fields - PowerOn will delete any data found in
+                                     USERCHAR12-USERCHAR15 of any unexpired
+                                     tracking type 8 records found on any 
+                                     open account.
+
+   Selection:
+    Any tracking type 8 record found on an open account.
+        
+   Sorting:
+    N/A
+    
+   Program Flow:
+    Program selects all open accounts with a tracking type 8 record. 
+
+        - PowerOn prints FM according to user input described above. 
+
+   Output:
+    "TRACKING 8 FM" - FM Report to be targeted in Miscellaneous
+                      Processing.
+    
+   Tips For Running 
+    Miscellaneous Processing Batch Program must be run afterward
+    to post the FM changes.
+]
+
+TARGET=TRACKING
+
+DEFINE
+ OPTIONSEL=NUMBER
+ OPTEXPIRE=0
+ OPTDELETE=1
+ OPTEDIT=2
+ DATENULL='--/--/--'
+END
+
+SETUP
+ OPTIONSEL=CODEREAD("Select Run Mode",
+                    "(0) Expire All Tracking 8s",
+                    "(1) Delete All Tracking 8s",
+                    "(2) Delete Banno-specific fields",
+                    "Input Run Type (0)default")
```

Comment:
Should you have a check here for those that can't read directions, to make sure they don't enter anything >2?

Reply:
This is a coderead, not a numberread. You can only choose what is offered when queuing up the program.